package.json

```diff
@@ -67,6 +68,7 @@
   },
   "peerDependencies": {
     "@zowe/cli": "^5.5.0",
+    "@zowe/cics": "^2.0.1",
```

Comment:
I'm interested to see what happens if you don't have the cics plugin installed.. I'll tinker with this. Think this bit is fine for now though.

Reply:
Discussed on Slack https://openmainframeproject.slack.com/archives/CC8AALGN6/p1559124222012100
Chris Boehm suggests using a regular dependency instead.